diff -urN bttv-0.7.78.orig/driver/Makefile bttv-0.7.78/driver/Makefile
--- bttv-0.7.78.orig/driver/Makefile	Mon Sep 10 21:06:48 2001
+++ bttv-0.7.78/driver/Makefile	Mon Sep 10 21:07:08 2001
@@ -3,8 +3,8 @@
 
 # where the kernel sources are located
 RUNNING := /lib/modules/$(shell uname -r)/build
-KERNEL_DIR := $(shell if test -d $(RUNNING); then echo $(RUNNING); else echo /usr/src/linux; fi)
-#KERNEL_DIR := /usr/src/linux
+#KERNEL_DIR := $(shell if test -d $(RUNNING); then echo $(RUNNING); else echo /usr/src/linux; fi)
+KERNEL_DIR := /usr/src/linux
 
 # kernel version
 CURRENT	:= $(shell ./uts-release $(KERNEL_DIR))
@@ -17,9 +17,7 @@
   KCOMPAT := kcompat24.o
 endif
 
-MODULE_DIR := $(shell test -d "/lib/modules/$(CURRENT)/kernel" && \
-	echo "/lib/modules/$(CURRENT)/kernel/drivers/media/video" ||\
-	echo "/lib/modules/$(CURRENT)/misc")
+MODULE_DIR := $(DESTDIR)/lib/modules/$(CURRENT)/misc
 
 EXTRA_CFLAGS=-g
 
@@ -59,7 +57,7 @@
 
 install:
 	strip --strip-debug *.o
-	su -c "set -x; mkdir -p $(MODULE_DIR); cp -v $(M_OBJS) $(MX_OBJS) $(MODULE_DIR); depmod -a"
+	set -x; mkdir -p $(MODULE_DIR); cp -v $(M_OBJS) $(MX_OBJS) $(MODULE_DIR)
 
 clean:
 	-rm -f *.o .*.o.flags *~
